app/src/main/java/com/example/oddhours/data/repository/JobRepository.kt

```diff
@@ -102,15 +114,34 @@ class JobRepository() {
          */
         for (job in jobModelList) {
             shiftsFromJobId = dbShifts.getShiftsForJobID(job.jobID)
+
+            var swap = true
```

Comment:
i was hoping we'd store the `shiftDayOfYear` variable and use it to sort in the ui simply by using a list sort method already available... that way we won't need to write our own function to sort it
 example: in `ParentAdapter` (~line 21) we're attaching the shifts list in reverse by using a list method. i was hoping we could utilize another list method to solve this, would look something like this:

`adapter = ChildAdapter(item.shifts.sortedByDescending { ShiftsModel -> ShiftsModel.shiftDayOfYear }, context, navController)`

i haven't tested this in code so syntax might be wrong but this site should help: https://www.codevscolor.com/kotlin-5-ways-sort-list-ascending-descending (the sortedByDescending() method)

Reply:
i tried this out on an online kotlin compiler, it seem to be working. I'll work on implementing it using that built in method.